lib/CL/devices/almaif/AlmaifDB/DBDevice.cc

```diff
@@ -0,0 +1,163 @@
+/* DBDevice.cc - Device based on parsing Almaif database and instantiating
```

Comment:
This comments makes no sense to an outside reader like me. What device? Is it an abstraction of a target device or something we control really?

Reply:
Added a comment there